### ISSUE TITLE
Feature request: UI Metric Plotter w/ Multiple Plot Lines

### ISSUE THREAD

**Issue by kaleko** [mlflow/mlflow#920]
I've logged several metrics (e.g. training loss during training epochs, CV loss during training epochs) and can view them as graphs individually in the tracking UI. I would like a feature that allows me to view them overlaid on the same axes in the UI, without exporting the data and plotting it myself

C/p from the slack chat:
mani   [1 day ago]
This is a great idea. Current design for the UI does not have a mechanism to choose multiple plot lines. To add this feature--there would be some design work needed. For now the easiest way, as you mentioned would be to use APIs to get metric history, and plot those.

mani   [1 day ago]
We will consider adding this feature. Please add a feature request

**Comment by erezsh** [comment]
Joining in, would like to see this as well.

**Comment by graffam** [comment]
Would love to see this as well!

**Comment by smurching** [comment]
This is now possible in MLflow 1.0 thanks to https://github.com/mlflow/mlflow/pull/1225 - if you compare multiple runs & click into a single metric, you can enter the names of additional metrics to visualize them all together

**Comment by smurching** [comment]
Closing this issue for now but feel free to reopen if anything comes up!

**Comment by Roffild** [comment]
On the page for comparing metrics of one run, there is not enough opportunity to add an additional plot.
This is necessary when the compared metrics have too different ranges of values.

For example:
first plot with a range of 1000, 5000
second plot with a range of 0.0, 1.0
